### DISCUSSION TITLE
AI tools (automatic segmentation) in 13.0 version are missing?

### DISCUSSION THREAD

**Discussion by ShiraHaskyAB** [cvat-ai/cvat#7865] [Q&A]
When I worked with the CVAT demo (12.1 version) the AI tools were present there. Now I installed the latest version (13.0), and the AI tools are missing. Is it possible? Should I need to install version 12.1 (as the demo) if I want the AI tools?

**Comment by bsekachev** [comment]
There were not any updates in 2.13 regarding AI tools.
Please, be sure you installation steps included steps to install serverless module. Refer to the installation documentation for details.
